build.gradle

```diff
@@ -33,21 +34,9 @@ allprojects {
 }
 
 apply plugin: 'idea'
-apply plugin: "com.diffplug.spotless"
 
-// Apply Spotless for buildSrc and build scripts
-spotless {
-    java {
-        target("buildSrc/**/*.java")
-        googleJavaFormat("1.17.0")
-    }
-    groovy {
-        target("buildSrc/**/*.groovy")
-    }
-    groovyGradle {
-        target('*.gradle', "**/*.gradle")
-    }
-}
+// apply SpotlessPlugin
+apply plugin: SpotlessPlugin
```

Comment:
> The latest CI failed because ctesque has Kotlin code. ctesque was not been checked by Spotless as it didn't include Spotless before, but this change in build.gradle will let Spotless to check Kotlin code in ctesque module, and the issue was spotted. So I think we can remove spotless configuration from submodules, and keep this one only. Looks like it can work in Robolectric repository.

Hii @utzcoz ..done all changes suggested by you...could you please review ?if everything is fine